backend/infrastructure/mongo/client_handler.py

```diff
@@ -0,0 +1,34 @@
+import gridfs
+from pymongo import MongoClient
+from typing import Optional
+from bson import ObjectId
+from fastapi import HTTPException
+
+client = MongoClient("mongodb://localhost:27017/")
```

Comment:
on client will be enough we should consime it in whole app.

Please provide interface (abstract class) with method signature and in this class implement them